tests/test_utils.py

```diff
@@ -32,23 +32,23 @@ def test_close_matches():
         'nouns/music_instruments',
     }
     # test specific group
-    assert randomname.util.close_matches('n/music') == [
-        'nouns/music_production', 'nouns/music_instruments', 'nouns/music_theory']
-    assert randomname.util.close_matches('nouns/music') == [
-        'nouns/music_production', 'nouns/music_instruments', 'nouns/music_theory']
-    assert randomname.util.close_matches('v/music') == [
-        'verbs/music', 'verbs/music_production']
-    assert randomname.util.close_matches('a/music') == ['adjectives/music_theory']
+    assert set(randomname.util.close_matches('n/music')) == {
```

Comment:
these ones made the CI fail

Reply:
I'm guessing some directory ordering thing. lgtm